gub/specs/lilypond-test.py

```diff
@@ -38,26 +38,35 @@ def install (self):
         self.system('''
 echo %(targetdir)s > %(builddir)s/input/regression/out-test/AAA-prefix
 ''')
+        # Exclusions: from share/lilypond/current we only need fonts,
+        # and we don't need their sources.
         self.system ('''
-LD_PRELOAD= tar -C %(builddir)s -cjf %(test_ball)s input/regression/out-test
+LD_PRELOAD= tar -C %(builddir)s -chjf %(test_ball)s --exclude 'current/[^f]*' --exclude 'fonts/source' input/regression/out-test out/share/lilypond/current
```

Comment:
My usual request: Please make lines shorter than 80 characters.

Reply:
I'm currently testing a change to call misc.join_lines() here, which I found used nearby.

Also, I notice that "out/share/lilypond/current" is still present from an earlier attempt and should not be necessary anymore.  I'm testing with that removed.

I'll force-push revised commits to this branch once my tests are complete (hopefully within a couple hours).